src/style.css

```diff
@@ -1 +1,83 @@
 /* add styles here */
+@font-face {
+  font-family: 'Avenir normal';
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Avenir bold';
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: bold;
+}
+
+body {
+  margin-top: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  flex-direction: column;
+  width: 100%;
+  display: flex;
+}
+
+.big-search {
+  margin-top: 20px;
+}
+
+.small-search {
+  margin-top: 20px;
```

Comment:
The spacing between the search bars is not implemented correctly. Please ensure the spacing matches the task requirements.